composer.json

```diff
@@ -22,13 +22,14 @@
     ],
     "require": {
         "php": ">=5.4",
-        "intervention/image": "~2.0",
+        "intervention/image": "~2.1",
         "league/flysystem": "~1.0",
         "symfony/http-foundation": "~2.3",
         "symfony/http-kernel": "~2.3"
     },
     "require-dev": {
         "mockery/mockery": "~0.9",
+        "phpunit/php-token-stream": ">=1.3.0",
```

Comment:
It's not really a good point, as the dependency is only relevant for phpunit, which has a bounded version constraint, so we don't care what version of php-token-stream we get, as long as it's above 1.3.0, and compatible with our ~4.4 of phpunit, which composer takes care of